CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -0,0 +1,116 @@
+# Changelog
+
+## [Unreleased](https://github.com/diem/diem/tree/HEAD)
+
+[Full changelog since diem-core-v1.1.5](https://github.com/diem/diem/compare/diem-core-v1.1.5...HEAD)
```

Comment:
Would we want to consider just logging changes since the last release branch rather than the last deployed release?  It might get a little odd if we deploy a hotfix release and then we have to go and update the change logs in main and the next release branch.

Reply:
Otherwise looks good.

Reply:
That could work. We will need to find hotfix release from their respective release branch. 

Maybe a bot can help so we can have something like k8s 
- in master https://github.com/kubernetes/kubernetes/commit/96a6df917434e7fce0d11bd62661260beb45e32d#diff-b8dffab3ab95835a7a4a1cc08ee5b557ee6be30f0aa9cba5b0769e66ad5d0916

- in release branch https://github.com/kubernetes/kubernetes/commit/2c03527428d49a709e807df4dfb3ca9d4bd6f679

Reply:
Updated to as suggested.